From b12ea8533123db01b3aa5ed9d972be7308eb20ee Mon Sep 17 00:00:00 2001
From: Enric Tejedor Saavedra <enric.tejedor.saavedra@cern.ch>
Date: Fri, 17 Feb 2017 14:57:30 +0100
Subject: [PATCH 2/7] Python3: __builtin__ does not exist, invoke built-in
 function __import__ directly

---
 bindings/pyroot/JupyROOT/kernel/utils.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/bindings/pyroot/JupyROOT/kernel/utils.py b/bindings/pyroot/JupyROOT/kernel/utils.py
index 300570c..25887ac 100644
--- a/bindings/pyroot/JupyROOT/kernel/utils.py
+++ b/bindings/pyroot/JupyROOT/kernel/utils.py
@@ -16,8 +16,6 @@ from JupyROOT.handlers import IOHandler, JupyROOTDeclarer, JupyROOTExecutor
 
 import ROOT
 
-import __builtin__
-
 _ioHandler = None
 _Executor  = None
 _Declarer  = None
@@ -48,7 +46,7 @@ class MagicLoader(object):
               if file != magics_path.replace("*.py","__init__.py"):
                   module_path="JupyROOT.kernel.magics."+file.split("/")[-1].replace(".py","")
                   try:
-                      module= __builtin__.__import__(module_path, globals(), locals(), ['register_magics'], -1)
+                      module= __import__(module_path, globals(), locals(), ['register_magics'], -1)
                       module.register_magics(kernel)
                   except ImportError:
                       raise Exception("Error importing Magic: %s"%module_path)
-- 
2.9.3

